src/ServiceControl.MSMQ.DLQMonitor/ServiceControl.MSMQ.DLQMonitor.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Should this project be in the new format or it's not supported by SC solution?

Reply:
Is it ok to mix and match multiple project formats in one solution? I see no reason why we couldn't use the new format here - unless the installer project which would pull this in wouldn't work with the new format?

Reply:
Even if it works I don't think we should mix project formats in the solution.